assets/grafana/celeborn-dashboard.json

```diff
@@ -1281,6 +1281,96 @@
           "title": "metrics_SlotsAllocated_increase_1h",
           "type": "timeseries"
         },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "palette-classic"
+              },
+              "custom": {
+                "axisCenteredZero": false,
+                "axisColorMode": "text",
+                "axisLabel": "",
+                "axisPlacement": "auto",
+                "barAlignment": 0,
+                "drawStyle": "line",
+                "fillOpacity": 0,
+                "gradientMode": "none",
+                "hideFrom": {
+                  "legend": false,
+                  "tooltip": false,
+                  "viz": false
+                },
+                "lineInterpolation": "linear",
+                "lineWidth": 1,
+                "pointSize": 5,
+                "scaleDistribution": {
+                  "type": "linear"
+                },
+                "showPoints": "auto",
+                "spanNulls": false,
+                "stacking": {
+                  "group": "A",
+                  "mode": "none"
+                },
+                "thresholdsStyle": {
+                  "mode": "off"
+                }
+              },
+              "mappings": [],
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green"
+                  },
+                  {
+                    "color": "red",
+                    "value": 80
+                  }
+                ]
+              }
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 8,
```

Comment:
Did you have validated the grafana dashboard?

Reply:
to be honest, I didn't as in pin I don't have a grafana environment , is there a doc telling how to validate in local?

Reply:
@CodingCat, you could use [Grafana Cloud](https://grafana.com/) to verify the dashboard, which refers to [Grafana Dashboard](https://stenicholas.grafana.net/goto/UZyL3oJIR?orgId=1).

Reply:
we'd better supply docs about how to verify, modify, and export the Grafana Dashboard with docker ...

Reply:
thanks @SteNicholas .... sorry for the delay...hectic week for me 

I tested in grafana cloud, while no data, but I think it should be fine

<img width="1596" alt="image" src="https://github.com/apache/celeborn/assets/678008/e51bde2f-579a-4e1f-aaef-763c511a15c2">